src/util/test/MetricTests.cpp

```diff
@@ -28,6 +28,31 @@ using uniform_u64 = stellar::uniform_int_distribution<uint64_t>;
 // how much data to keep in memory when comparing datasets
 static std::chrono::seconds const sampleCutoff(60 * 5);
 
+void
+sleepTillNextBucketIfNecessary()
```

Comment:
nit: this should have `windowSize` as argument instead of hard coding 1 second

Reply:
Fixed.